spec.html

```diff
@@ -6888,6 +6909,32 @@ <h1>
           1. Set _k_ to _k_ + 1.
       </emu-alg>
     </emu-clause>
+
+    <emu-clause id="sec-SetterThatIgnoresPrototypeProperties" type="abstract operation">
+      <h1>
+        SetterThatIgnoresPrototypeProperties (
+          _this_: an ECMAScript language value,
```

Comment:
```suggestion
          _thisValue_: an ECMAScript language value,
```

Given that we say things like "the this value", it'd be best to not have an alias with the same name.